lib/plugins/threaded-signing/plugin.c

```diff
@@ -30,20 +30,25 @@
 #include <glib.h>
 #include <stdlib.h>  // calloc, malloc, free
 #include <string.h>  // memcpy
+#include <unistd.h>
 
 #include "includes/signed_video_interfaces.h"
 #include "includes/signed_video_openssl.h"
 
+#define MAX_HASHES 5
+typedef struct _threaded_signature_info_t threaded_signature_info_t;
+
 typedef enum {
-  THREADED_SIGNING_WAITS_FOR_HASH_TO_SIGN,
-  THREADED_SIGNING_HAS_HASH_TO_SIGN,
-  THREADED_SIGNING_SIGNS_HASH,
-  THREADED_SIGNING_HAS_SIGNATURE,
-  THREADED_SIGNING_ERROR,
+  THREADED_SIGNING_ERROR = 1,
 } threaded_signing_plugin_state;
```

Comment:
I think you should try to get rid of this enum. It has no meaning now when there is only one value.